spec/features/user_visits_homepage_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.feature "User visits homepage" do
+  scenario "sees list of redirections" do
+    visit root_path
+
+    expect(page.html).to have_css("table.redirections")
```

Comment:
This line feels more like it belongs in a view spec -- it doesn't actually break if you remove this, right? I think the `within` will break if this isn't present, too.

What do you think about removing this line and keeping the rest of the spec as-is